packages/sdk/src/types/lend.ts

```diff
@@ -51,7 +51,7 @@ export interface TransactionData {
   /** Encoded function call data */
   data: Hex
   /** ETH value to send */
-  value: bigint
+  value: bigint | Hex
```

Comment:
🤔 This feels easy to mess up. I'd rather we just switched all instances to Hex.

Reply:
I believe viem accepts both types as valid values and under the hood viem will encode it for you, but I'm down to just leave it as `bigint`, will update